Python/correlation/correlation.py

```diff
@@ -0,0 +1,59 @@
+"""Provides functions to correlate signals.
+"""
+
+import numpy as np
+import math
+from scipy import signal
+
+import Python.signal_generation.positioning as sg_pos
+
+
+def give_correlation_matrix(
```

Comment:
```suggestion
def calculate_correlation_matrix(
```
or `compute` :man_shrugging: